core/zoom_controls.js

```diff
@@ -251,3 +157,144 @@ Blockly.ZoomControls.prototype.position = function() {
   this.svgGroup_.setAttribute('transform',
       'translate(' + this.left_ + ',' + this.top_ + ')');
 };
+
+/**
+ * Create the zoom in icon and its event handler.
+ * @param {string} rnd The random string to use to identify the clip paths.
```

Comment:
Maybe pass in a complete id, instead of a id suffix? At least clarify the comment to note this is a suffix appended to the id.

It's not clear to me why the id needs to be randomized in the first place.

Reply:
https://github.com/google/blockly/blob/master/core/inject.js#L118

Reply:
Added comments.